backend/graminsta/post/models.py

```diff
@@ -0,0 +1,29 @@
+"""
+Here defines the database tables for post app
+"""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Comment(models.Model):
+    """
+    Comment Table
+    """
+    commentId = models.IntegerField(default=0)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE)
+    time = models.DateTimeField(auto_now=True)
+
+
+class Post(models.Model):
+    """
+    Post Table
+    """
+    postId = models.IntegerField(default=0)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE,
+                                  related_name="Publisher_Of_Post")
+    time = models.DateTimeField(auto_now=True)
+    markedUser = models.ManyToManyField(
+        User, related_name="User_Who_Marked_Post")
+    content = models.BinaryField()
+    comments = models.ForeignKey(Comment, on_delete=models.CASCADE)
```

Comment:
According to the doc, class ForeignKey(to, on_delete, **options) is defined as "A many-to-one relationship", so I guess maybe it should be like this?
```python
class Comment:
    post = models.ForeignKey(Post, on_delete=models.CASCADE)
```

Reply:
Sounds reasonable!